README.md

```diff
@@ -356,3 +357,25 @@ $app
             );
         });
 ```
+
+#### Tput
+
+The tput helper will help you get informed about user's terminal capabilities. The helper provides
+a single entry point to all capabilities: the `get` method. Here is how you would do to get the `clear_screen`
+capability:
```

Comment:
The `get` method falls back on ˋcount` or ˋhas`. This was done to make the API simple. 
The user can still use native methods: the helper extends the original Tpit class.